### ISSUE TITLE
Primal does not work on Android 13

### ISSUE THREAD

**Issue by DiegoYegros** [PrimalHQ/primal-android-app#271]
I can't open the app. It crashes.

I downloaded the Play Store version, 20.0.34.
My device is a POCO F3, running HyperOS 1.0.3.0, Android 13.

### Steps to reproduce
Download the latest Primal app on Android 13 and open it. It crashes.

### Additional context
I've also tested on the Android Studio simulator with a Pixel 6 API 33, and the app crashes starting from version 2.0.30.
Version 2.0.29 opens correctly.
I'm not an Android developer, but if it's of any help, this is what I see in Logcat:
```
01:57:06.143  E  FATAL EXCEPTION: main
                 Process: net.primal.android, PID: 4656
                 java.lang.IllegalArgumentException: Invalid ID, must be in the range [0..16)
                 	at android.graphics.ColorSpace.get(ColorSpace.java:1390)
                 	at android.graphics.Color.colorSpace(Color.java:703)
                 	at android.graphics.Paint.setColor(Paint.java:1120)
                 	at androidx.compose.ui.graphics.WrapperVerificationHelperMethods.setColor-4WTKRHQ(AndroidPaint.android.kt:281)
                 	at androidx.compose.ui.graphics.AndroidPaint_androidKt.setNativeColor-4WTKRHQ(AndroidPaint.android.kt:171)
                 	at androidx.compose.ui.graphics.AndroidPaint.setColor-8_81llA(AndroidPaint.android.kt:63)
                 	at androidx.compose.ui.graphics.SolidColor.applyTo-Pq9zytI(Brush.kt:406)

```

**Comment by markocic** [comment]
Thank you for the very detailed additional context! I was able to track down the commit that introduced this crash and it should be fixed soon.

**Comment by AleksandarIlic** [comment]
The fix will be released soon in 2.0.35+ build.
